integr8ly/integreatly-operator
# Intly 4025

Add blackbox targets for all products (excluding MSS and MDC).

Verification steps:
1. Run a full installation from this branch
2. Check that blackbox targets for each product are created in the monitoring namespaces
3. Check that all blackbox targets are reconciled into the additional scrape config secret
